spec/spec_helper.rb

```diff
@@ -8,6 +9,8 @@
   # Disable RSpec exposing methods globally on `Module` and `main`
   config.disable_monkey_patching!
 
+  config.expose_dsl_globally = true
```

Comment:
What does this do? Lets add a comment?

Reply:
This lets us say `describe` instead of `Rspec.describe`; not really essential, but it's the style we use elsewhere. Will comment.